samples/app/src/main/java/com/example/app/MainActivity.kt

```diff
@@ -11,10 +11,7 @@ import android.os.Bundle
 import androidx.activity.compose.setContent
 import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
-import androidx.compose.runtime.collectAsState
-import androidx.compose.runtime.getValue
 import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
-import org.forgerock.android.auth.FRUserKeys
 
 class MainActivity : AppCompatActivity() {
```

Comment:
For fully compose app which has only one ComponentActivity but does not have AppCompatActivity or FragmentActivity or fragments . is there a way to invoke this FRuser.browser.login() ?